web/themes/joinup/joinup_theme.theme

```diff
@@ -193,6 +192,7 @@ function joinup_theme_preprocess_block(&$variables) {
       case 'view.collections.page_1':
       case 'view.solutions.page_1':
       case 'view.content_overview.page_1':
+      case 'entity.rdf_entity.member_overview':
```

Comment:
Small problem with height of image when the tile is used on the user profile:
![members joinup](https://user-images.githubusercontent.com/1923476/26878431-b45664c4-4b8d-11e7-82f7-25bfb63a25f4.png)

Without the height property:
![members joinup 1](https://user-images.githubusercontent.com/1923476/26878436-be807ab6-4b8d-11e7-93ae-42059361141d.png)